src/data.ts

```diff
@@ -0,0 +1,34 @@
+
+
+const data = {
+    contactInfo: {
+        fullName: "Oscar Enrique Muñoz Lima",
+        emailAddress: "cok.kike@gmail.com",
+    },
+    github: {
+        profileURL: "https://github.com/cokkike88",
+        username: "cokkike88",
+    },
+    credentials: {
+        password: process.env.PASS,
+    },
+    personalInfo: {
+        questions: [
+            {
+                question: "If I was a Sr. Programmer, I would like to build:",
+                answer: "Respuesta: Some application with micriservices with aws lambda, with neptuno database, nodejs as backend, reactjs as frontend, aws cognito for authentication and authorization, aws s3 to resources storage, aws api gateway",
```

Comment:
hay que poner atención a los detalles

```suggestion
                answer: "Respuesta: Some application with microservices with aws lambda, with neptuno database, nodejs as backend, reactjs as frontend, aws cognito for authentication and authorization, aws s3 to resources storage, aws api gateway",
```